students/RimleeT/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,92 @@
+"""
+Write some functions that take a sequence as an argument, and return a copy of that sequence:
+with the first and last items exchanged.
+with every other item removed.
+with the first 4 and the last 4 items removed, and then every other item in the remaining sequence.
+with the elements reversed (just with slicing).
+with the last third, then first third, then the middle third in the new order.
+"""
+
+# Exchange first and last item
+def exchange_first_last(seq):
+	if len(seq)<=1:
+		return seq
+	else:
+		a_new_sequence = seq[-1]+ seq[1:len(seq)-1] + seq[0]
+		return a_new_sequence
+	
+assert exchange_first_last("this is a string") == "ghis is a strint"
+assert exchange_first_last("w") == "w"
+
+# Keep first and last items with other items removed. 
+def only_first_last(seq):
+    if len(seq)<=1:
+        return(seq)
+    else:
+        a_new_sequence = seq[0]+seq[-1]
+        return(a_new_sequence)
+
+assert only_first_last("this is a string") == "tg"
+assert only_first_last("w") == "w"
+
+#first 4 and the last 4 items removed
+def first_and_last_four_removed (seq):
+	if len(seq)<=8:
+		#print(seq)
+		return(seq)
+	else:
+		a_new_sequence = seq[4:]
+		a_new_sequence = a_new_sequence[:-4]
+		#print(a_new_sequence)
+		return(a_new_sequence)
+		
+assert first_and_last_four_removed("this is a string") == " is a st"
+assert first_and_last_four_removed("w") == "w"
+assert first_and_last_four_removed("abcd") == "abcd"
+
+#elements reversed
+def elements_reversed (seq):
+	if len(seq)<=1:
```

Comment:
do you need to check this? will the slice still work with a one element sequence?